pom.xml

```diff
@@ -121,7 +122,12 @@
   </build>
 
   <dependencies>
-    <dependency>
+   <dependency>
+     <groupId>org.jenkins-ci.plugins</groupId>
+     <artifactId>structs</artifactId>
```

Comment:
what's the purpose to adding this dependency?  if there is one could you include a comment in the commit message?